examples/mfa_hierarchy/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# build/run \
```

Comment:
remove commented lines

Reply:
+1

Reply:
Do you want us to leave this new folder we are using to test the new hierarchy in the final merge?